test/testoffscreen.c

```diff
@@ -113,23 +113,23 @@ int main(int argc, char *argv[])
     if (!SDL_InitSubSystem(SDL_INIT_VIDEO)) {
         SDL_Log("Couldn't initialize the offscreen video driver: %s\n",
                 SDL_GetError());
-        return SDL_FALSE;
+        return 0;
```

Comment:
On failure, it's common to return a non-zero value.

Reply:
> On failure, it's common to return a non-zero value.

oops sry

Reply:
No need to say sorry. This was wrong before.

Reply:
> No need to say sorry. This was wrong before.

Yes but a warning would have been better in this case to indicate that something is wrong.
Should have read the code more carefully.
<3